helm/agents/observability/values.yaml

```diff
@@ -1,10 +1 @@
 modelConfigRef: ""
-prometheus:
-  url: "prometheus.kagent:9090/api/v1"
-  username: ""
-  password: ""
```

Comment:
All of this was unused so cleaned up while I was at it. Same goes for `grafana.username` and `grafana.password` just below.